### ISSUE TITLE
double affiliations not implemented

### ISSUE THREAD

**Issue by rzehumat** [rzehumat/cygMikulas#3]

**Comment by rzehumat** [comment]
The current implementation takes a label. It should be changed to a list of labels (if lists exist in LaTeX) or to list comma-separated commands, e.g. one of these
```latex
\name[*]{Jára Cimrman}{fvu,nerc}
\name[*]{Jára Cimrman}{\inst{fvu},\inst{nerc}}
```

**Comment by rzehumat** [comment]
Until we come up with something better, one could implement `\oneInst` and `\twoInst` (assuming not many people have 3 or more affiliations).

**Comment by rzehumat** [comment]
Closed by 44c8fa5.
One had to use `cleveref` package with `\crefname` and `\crefmultiformat`.
